src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -701,7 +701,8 @@ private MemberSemanticModel GetMemberModel(int position)
                         var constructorDecl = (ConstructorDeclarationSyntax)memberDecl;
                         outsideMemberDecl =
                             !LookupPosition.IsInConstructorParameterScope(position, constructorDecl) &&
-                            !LookupPosition.IsInParameterList(position, constructorDecl);
+                            !LookupPosition.IsInParameterList(position, constructorDecl) &&
+                            !LookupPosition.IsInExpressionBody(position, constructorDecl.GetExpressionBodySyntax(), constructorDecl.SemicolonToken);
```

Comment:
Also, It looks like the previous switch section, for accessors, should handle expression-bodied declarations. I would say we should add LookupPosition.IsInBody overload for an AccessorDeclarationSyntax, similar to the one for BaseMethodDeclarationSyntax.